bcda/cclf/fileprocessor.go

```diff
@@ -113,8 +113,8 @@ func (p *processor) handleArchiveError(path string, info os.FileInfo, cause erro
 }
 
 func getCMSID(name string) (string, error) {
-	// CCLF foldername convention with BCD identifier: P.BCD.<ACO_ID>.ZCY**.Dyymmdd.Thhmmsst
-	exp := regexp.MustCompile(`(?:T|P)\.BCD\.(.*)\.ZCY\d{2}\.D\d{6}\.T\d{7}`)
+	// CCLF foldername convention with BCD identifier: P.BCD.<ACO_ID>.ZC[Y|R]**.Dyymmdd.Thhmmsst
+	exp := regexp.MustCompile(`(?:T|P)\.BCD\.(.*)\.ZC(?:Y|R)\d{2}\.D\d{6}\.T\d{7}`)
```

Comment:
Should this be `[Y|R]` instead of `?:Y|R`?

Reply:
Was trying to figure out a non-capturing OR expression!